src/pregh-changes.md

```diff
@@ -7,39 +7,39 @@
 
 ## [1.1.0](https://doi.org/10.5281/zenodo.3759802) (2018-04-19)
 
--   Added support for MEG data (merged BEP008)
+-   Added support for MEG data (merged BEP008).
 -   Added `SequenceName` field.
--   Added support for describing events with Hierarchical Event Descriptors [[4.3 Task events](04-modality-specific-files/05-task-events.md)].
--   Added `VolumeTiming` and `AcquisitionDuration` fields [[4.1 Task (including resting state) imaging data](04-modality-specific-files/01-magnetic-resonance-imaging-data.md#task-including-resting-state-imaging-data)].
+-   Added support for describing events with Hierarchical Event Descriptors: [4.3 Task events](04-modality-specific-files/05-task-events.md).
+-   Added `VolumeTiming` and `AcquisitionDuration` fields: [4.1 Task (including resting state) imaging data](04-modality-specific-files/01-magnetic-resonance-imaging-data.md#task-including-resting-state-imaging-data).
```

Comment:
This style change from `[[.....]]` to `: [...]` is also for convenience of link parsing.